### PR TITLE
chore(css): removed the usage of .odd and .even classes

### LINKED ISSUE #7049: Deprecate 'odd' and 'even' classes
Checking odd/even rows/items when constructing tables and lists makes little sense. This should be done via css pseudo selectors.

### PR DESCRIPTION
**Pull request by jdalsem**
fixes #7049

### CODE REVIEW

File: mod/groups/views/default/groups/profile/fields.php [hunk 1]
```diff
@@ -26,13 +24,11 @@
 			$options['tag_names'] = $key;
 		}
 
-		echo "<div class=\"{$even_odd}\">";
+		echo "<div>";
```
Comment (hypeJunction) [inline mod/groups/views/default/groups/profile/fields.php]: Add a view to core and have both profile and groups plugin use it.

File: mod/profile/views/default/profile/css.php [hunk 2]
```diff
@@ -59,12 +59,7 @@
 #profile-details .wire-status {
 	margin-top: 10px;
 }
-#profile-details .odd {
-	border-bottom: 1px solid #DCDCDC;
-	margin: 0;
-	padding: 5px 0;
-}
-#profile-details .even {
+.profile-fields > div {
```
Comment (hypeJunction) [inline mod/profile/views/default/profile/css.php]: Move these to core, add an actual class to the div and have both gruops and profile use it.

**Review by hypeJunction — DISMISSED** [review]

File: mod/profile/views/default/profile/css.php (hunk 2 above)
Comment (jdalsem) [inline mod/profile/views/default/profile/css.php]: can also do a .elgg-divider-bottom + .pvs then there will be no need for the extra div and classes

Comment (hypeJunction) [inline mod/profile/views/default/profile/css.php]: I am killing .p* and .m* classes. Just use CSS.

Comment (jdalsem) [inline mod/profile/views/default/profile/css.php]: added a issue for reworking the profile field listings on profile/group https://github.com/Elgg/Elgg/issues/10412 so we can keep it separate from this PR

**Comment by jdalsem** [comment]
@hypeJunction could this go in as i create a rework ticket for the group/profile profile fields?

**Pull request merged**